control/narrative_section.go

```diff
@@ -58,7 +60,11 @@ func (n narrativeSection) Fill(data opencontrols.Data, control string) (err erro
 		return
 	}
 
+	// fixup the narrative
 	narrative := data.GetNarrative(control, key)
+	if strings.Contains(control, "AU-2") {
+		logger.Debugf("'%s': '%s': %v", control, key, narrative)
+	}
```

Comment:
I assume this was just for debugging - mind removing this and the similar lines in other files?